### PR TITLE
Feature/integration

### PR DESCRIPTION
**Pull request by jonathanduperrier**
adding travis integration

### CODE REVIEW

**Comment by jonathanduperrier** [comment]
Adding travis integration in neuromorphic platform

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+language: python
+python:
+- 2.7
+- 3.5
+env:
+- DJANGO_VERSION=1.9.6
+install:
+- pip install pip-tools
+- pip-sync deployment/requirements-deploy.txt
+- pip download -i https://bbpteam.epfl.ch/repository/devpi/simple --pre -r deployment/requirements-bbp.txt
```
Comment (apdavison) [inline .travis.yml]: I think this line can be removed

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- pip install psycopg2
+- pip install numpy
+before_script:
+- export PYTHONPATH=$PYTHONPATH:$(pwd)
+- python -c "import django; print(django.get_version())"
+- export NMPI_ENV=travis
+script:
+# - sleep 1m
+- cd job_manager
+- python manage.py migrate --noinput
```
Comment (apdavison) [inline .travis.yml]: This should be `python manage.py test simqueue`

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+# - python manage.py test simqueue
+- python manage.py runserver 0.0.0.0:8000 &
```
Comment (apdavison) [inline .travis.yml]: This line is not needed

File: hbp-service-client/.gitignore [hunk 4]
```diff
@@ -0,0 +1,61 @@
+# Byte-compiled / optimized / DLL files
```
Comment (apdavison) [inline hbp-service-client/.gitignore]: `hbp-service-client` should not be included in the pull request, it should be installed using pip

File: job_manager/requirements.txt [hunk 5]
```diff
@@ -2,10 +2,14 @@
 # Autogenerated by pip-compile then hand-edited to make local paths relative
 #
 
--e packages/task-types-0.0.10
--e packages/bbp-services-0.0.14
--e packages/hbp-app-python-auth-0.0.7
--e packages/bbp-client-0.3.18
+#-e packages/task-types-0.0.10
```
Comment (apdavison) [inline job_manager/requirements.txt]: Commented-out code should be deleted.

File: simqueue/views.py [hunk 6]
```diff
@@ -110,9 +110,12 @@ def copy_datafiles_to_storage(request, target, job_id):
 def copy_datafiles_to_collab_storage(request, job, local_dir, relative_paths):
 
     # upload local files to collab storage
-    from bbp_client.oidc.client import BBPOIDCClient
-    from bbp_client.document_service.client import Client as DocClient
-    import bbp_services.client as bsc
+    #from bbp_client.oidc.client import BBPOIDCClient
```
Comment (apdavison) [inline simqueue/views.py]: There seem to be some missing changes here - i.e. the changes from the `feature/without-epfl-modules` branch
